maestrowf/maestro.py

```diff
@@ -61,17 +62,43 @@
 
 def status_study(args):
     """Check and print the status of an executing study."""
-    status = Conductor.get_status(args.directory)
-
-    if status:
-        print(tabulate.tabulate(status, headers="keys"))
-        return 0
+    #args_dir is the path specified
+    args_dir = str(args.directory)
+    args_dir = args_dir.replace("[", "")
+    args_dir = args_dir.replace("]", "")
+    args_dir = args_dir.replace("'", "")
+
+    #Get a list of all the directories based on the path
+    directory_list = []
```

Comment:
You don't need to initialize the `directory_list` before using it -- just assign it like you do one `Line 73`